### PR TITLE
mmc-utils: 2015-11-18 -> 2018-03-27

### PR DESCRIPTION
**Pull request by dmmarkov**
###### Motivation for this change
This is package is very old. I need new version of it in my product so I decide to update it.

###### Things done

- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by Mic92** [comment]
@GrahamcOfBorg build mmc-utils

File: pkgs/os-specific/linux/mmc-utils/default.nix [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
 
   src = fetchgit {
     url = "git://git.kernel.org/pub/scm/linux/kernel/git/cjb/mmc-utils.git";
-    rev = "44f94b925894577f9ffcf2c418dd013a5e582648";
-    sha256 = "0hkdzc71pdnscbpdpgwljcchiyancarldjyd0w609sy18bky833x";
+    rev = "b4fe0c8c0e57a74c01755fa9362703b60d7ee49d";
+    sha256 = "01llwan5j40mv5p867f31lm87qh0hcyhy892say60y5pxc0mzpyn";
   };
 
   makeFlags = "CC=${stdenv.cc.targetPrefix}cc";
```
Comment (Mic92) [inline pkgs/os-specific/linux/mmc-utils/default.nix]: it is a bit weird though, that it does not use `makeFlags = "CC=cc";` instead.

**Comment by arch1tect0r** [comment]
@dezgeg

File: pkgs/os-specific/linux/mmc-utils/default.nix (hunk 1 above)
Comment (Mic92) [reply pkgs/os-specific/linux/mmc-utils/default.nix]: @bgamari is `CC=cc` not best practice?

Comment (bgamari) [reply pkgs/os-specific/linux/mmc-utils/default.nix]: IIRC `cc` may not exist when cross compiling.

**Pull request merged**